### ISSUE TITLE
Write accessibility acceptance criteria for components

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-design-system#1848]
## What
Write accessibility acceptance criteria for all our components. 

## Why
We are currently [writing some documentation on accessibility acceptance criteria](https://github.com/alphagov/govuk-design-system/issues/1781) that 
- defines general accessibility acceptance criteria that all components should meet
- explains how to write component specific accessibility acceptance criteria

Once published, we should write component specific accessibility acceptance criteria for all our components as it will help us to check for the correct behaviour when iterating the components in the future and to catch regressions.

## Who needs to know about this
Developers, tech writer

## Done when

- [x] Agreed where the component accessibility acceptance criteria should be published and in what format
- [x] Written accessibility acceptance criteria for our components (or created smaller cards to split this work)
- [ ] Tested our components against the new component specific criteria, as well as the more general accessibility acceptance criteria
- [ ] Published the accessibility acceptance criteria for components
